packages/cpp/src/common/cpp-preferences.ts

```diff
@@ -1,65 +0,0 @@
-/*
- * Copyright (C) 2017 Ericsson and others.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
- */
-
-import { interfaces } from "inversify";
-import {
-    createPreferenceProxy,
-    PreferenceProxy,
-    PreferenceService,
-    PreferenceContribution,
-    PreferenceSchema
-} from '@theia/preferences/lib/common';
-
-export const CLANGD_COMMAND_DEFAULT = 'clangd';
-
-export const CppConfigSchema: PreferenceSchema = {
```

Comment:
please assign someone who can review cpp extension

Reply:
I've asked @MarkZ3 and for now I think it's okay to remove them and replace with env variables as we're still not sure how we're going to configure the backend (seems like it should be handled by something else than the frontend preferences)...